examples/connectivity/plot_pac.py

```diff
@@ -0,0 +1,183 @@
+# time windows. In this example, we'll simulate two signals. One
+# of the signals has an amplitude that is locked to the phase of
+# the other. We'll calculate PAC for a number of time points, and
+# in both directions to show how PAC responds.
+
+# Define parameters for our simulated signal
+sfreq = 1000.
+f_phase = 5
+f_amp = 40
+frac_pac = .99  # This is the fraction of PAC to use
+mag_ph = 4
+mag_am = 1
+
+# These are the times where PAC is active in our simulated signal
+n_secs = 20.
+time = np.arange(0, n_secs, 1. / sfreq)
+event_times = np.arange(1, 18, 4)
+event_dur = 2.
+
+# Create a time mask that defines when PAC is active
+msk_pac_times = np.zeros_like(time).astype(bool)
+for i_time in event_times:
+    msk_pac_times += mne.utils._time_mask(time, i_time, i_time + event_dur)
+
+# Now simulate two signals. First, a low-frequency phase
+# that modulates high-frequency amplitude
+_, lo_pac, hi_pac = simulate_pac_signal(time, f_phase, f_amp, mag_ph, mag_am,
+                                        frac_pac=frac_pac,
+                                        mask_pac_times=msk_pac_times)
+
+# Now two signals with no relationship between them
+_, lo_none, hi_none = simulate_pac_signal(time, f_phase, f_amp, mag_ph,
+                                          mag_am, frac_pac=0,
+                                          mask_pac_times=msk_pac_times)
+
+# Finally we'll mix them up.
+# The low-frequency phase of signal A...
+signal_a = lo_pac + hi_none
+# Modulates the high-frequency amplitude of signal B. But not the reverse.
+signal_b = lo_none + hi_pac
+
+
+# We'll visualize these signals. A on the left, B on the right
+# The top row is a combination of the middle and bottom row
+labels = ['Combined Signal', 'Lo-Freq signal', 'Hi-freq signal']
+data = [[signal_a, lo_none, hi_pac],
+        [signal_b, lo_pac, hi_none]]
+fig, axs = plt.subplots(3, 2, figsize=(10, 5))
+for axcol, i_data in zip(axs.T, data):
+    for ax, i_sig, i_label in zip(axcol, i_data, labels):
+        ax.plot(time, i_sig)
+        ax.set_title(i_label, fontsize=20)
+_ = plt.setp(axs, xlim=[8, 12])
+plt.tight_layout()
+
+# Create a raw array from the simulated data
+info = mne.create_info(['pac_hi', 'pac_lo'], sfreq, 'eeg')
+raw = mne.io.RawArray([signal_a, signal_b], info)
+
+# The PAC function needs a lower and upper bound for each frequency
+f_phase_bound = (f_phase-.1, f_phase+.1)
+f_amp_bound = (f_amp-2, f_amp+2)
+
+# First we'll calculate PAC for the entire timeseries.
+# We'll use a few PAC metrics to compare.
+iter_pac_funcs = [['glm', 'ozkurt'], ['plv']]
+win_size = 1.  # In seconds
+step_size = .1
+pac_times = np.array(
+    [(i, i + win_size)
+     for i in np.arange(0, np.max(time) - win_size, step_size)])
+
+# Here we specify indices to calculate PAC in both directions
+ixs = np.array([[0, 1],
+                [1, 0]])
+fig, axs = plt.subplots(2, 1, figsize=(10, 5))
+all_pac = []
+for pac_funcs in iter_pac_funcs:
+    pac, pac_freqs = phase_amplitude_coupling(
```

Comment:
This particular warning uses both `warnings.warn()` and `logger.warning()` (no idea why), so you have to silence both:
you should add in plot.pac.py:
```python
import logging
logger = logging.getLogger('mne')
logger.setLevel(50)
```